src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1269,7 +1266,28 @@ static bool addTriggeredSubscriptions_noCache
         std::string geometry = expr.hasField(CSUB_EXPR_GEOM)   ? getStringFieldF(expr, CSUB_EXPR_GEOM)   : "";
         std::string coords   = expr.hasField(CSUB_EXPR_COORDS) ? getStringFieldF(expr, CSUB_EXPR_COORDS) : "";
 
-        trigs->fillExpression(q, georel, geometry, coords);
+        trigs->fillExpression(georel, geometry, coords);
+
+        // Parsing q
+        if (q != "")
+        {
+          StringFilter* stringFilterP = new StringFilter();
+
+          if (stringFilterP->parse(q.c_str(), &err) == false)
+          {
+            delete stringFilterP;
+          
+            LM_E(("Runtime Error (%s)", err.c_str()));
+            TIME_STAT_MONGO_READ_WAIT_STOP();
```

Comment:
You mean it has been removed and then put back again?
Perhaps via one of the many merges with develop ...
I have no problem in removing it (again?), no reason to measure times in failed requests, I guess ...